YAXLib/YAXExceptions.cs

```diff
@@ -549,15 +690,15 @@ public class YAXBadlyFormedXML : YAXException
         /// Initializes a new instance of the <see cref="YAXBadlyFormedXML"/> class.
         /// </summary>
         /// <param name="innerException">The inner exception.</param>
-        public YAXBadlyFormedXML(Exception innerException)

```

Comment:
Yes, good point. I'd initially looked at it a bit too briefly and not bothered to change it because it didn't have an access to an object that implemented IXmlLineInfo to pull the line info from. However, it turns out the the XmlException class has a LineNumber and LinePosition that I could use, so that works too now.